rustc_codegen_spirv/src/symbols.rs

```diff
@@ -279,15 +414,169 @@ pub fn parse_attrs(
+                                                                }
+                                                            },*/
+                                                            _ => {
+                                                                if let Some(val) = val {
+                                                                    e.execution_modes.push((*execution_mode, ExecutionModeExtra::new([val])));
+                                                                }
+                                                                else {
+                                                                    e.execution_modes.push((*execution_mode, ExecutionModeExtra::new([])));
+                                                                }
+                                                            }
+                                                        }
+                                                    }
+                                                    else {
+                                                        cx.tcx.sess.span_err(
+                                                            arg.span(),
+                                                            &format!(
+                                                                "#[spirv({}(..))] unknown attribute argument {}", 
+                                                                name.name.to_ident_string(),
+                                                                attr_name.name.to_ident_string()
+                                                            ),
+                                                        );
+                                                    }
+                                                }
+                                                else {
+                                                    cx.tcx.sess.span_err(
+                                                        arg.span(),
+                                                        &format!(
+                                                            "#[spirv({}(..))] attribute argument must be single identifier", 
+                                                            name.name.to_ident_string()
+                                                        ),
+                                                    );
+                                                }
+                                                    
+                                            });
+                                    }
+                                    match e.execution_model {
+                                        Fragment => {
+                                            //FIXME: Is this the correct default?
+                                            let origin_mode = origin_mode.unwrap_or(OriginUpperLeft);
+                                            e.execution_modes.push((origin_mode, ExecutionModeExtra::new([])));
```

Comment:
This is a little convoluted code so I might be missing something, but it might be nice to only specify this if the user didn't do so themselves. (Considering we didn't even allow users to specify it before this PR, this is fine as-is for now, mostly just making a note that we should follow up on this later!)

Reply:
This isn't a breaking change. The origin_mode variable is None, and replaced with OriginUpperLeft or OriginLowerLeft if specified. Before it was hardcoded in entry.rs for fragment shaders.